mbtiles/src/bin/mbtiles.rs

```diff
@@ -33,11 +35,26 @@ pub struct Args {
     command: Commands,
 }
 
+#[derive(
+    Default, Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize, EnumDisplay, ValueEnum,
+)]
+#[enum_display(case = "Kebab")]
+pub enum OutputFormat {
+    #[default]
+    Text,
+    Json,
+    JsonPretty,
```

Comment:
Because people will misremember

```suggestion
    #[value(alias("pretty-json"))]
    JsonPretty,
```